internal/super/setup.go

```diff
@@ -0,0 +1,181 @@
+package super
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/go-git/go-git/v5"
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowkit"
+	"github.com/onflow/flow-cli/pkg/flowkit/output"
+	"github.com/onflow/flow-cli/pkg/flowkit/services"
+	"github.com/spf13/cobra"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+)
+
+type FlagsSetup struct {
+	Scaffold bool `default:"" flag:"scaffold" info:"Use provided scaffolds for project creation"`
+}
+
+var setupFlags = FlagsSetup{}
+
+var SetupCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "setup <path>",
+		Short:   "Start a new Flow project",
+		Example: "flow setup my-project",
+		Args:    cobra.ExactArgs(1),
+		GroupID: "super",
+	},
+	Flags: &setupFlags,
+	Run:   create,
+}
+
+const scaffoldListURL = "https://raw.githubusercontent.com/onflow/flow-scaffold-list/main/scaffold-list.json"
```

Comment:
I'm a bit worried to take this from `/main/` is there any way it can be from a tag instead?

Reply:
I can see what you mean, but my thinking is: main is protected, so any changes to that list mush be approved, and that is not the main reason, which is, I want to make this accessible for anyone to add to, and if we also require a tag then they will have to first make a PR which gets merged and then ask us to tag it and then make another PR to update the version here, so a bit of extra friction. The thing tho, which is here not visible is that I plan to close that repo and just have the list in the CLI repo. I plan to fix that today.